src/pages/about-us/team/index.tsx

```diff
@@ -1,13 +1,45 @@
 // path: about-us/team/leadership
 
 import { Typography } from '@mui/material';
+import { GetServerSideProps } from 'next';
 
-const TeamLeadershipPage = () => {
+// import { TeamCoreData, TeamContactData, TeamMembersData } from '@utils/types';
+import { TeamApiResponse } from '@utils/types';
+import { fetchData } from 'lib/api';
+
+interface TeamPageProps {
+  team: TeamApiResponse;
+}
+
+const TeamLeadershipPage = ({ team }: TeamPageProps) => {
```

Comment:
would be nice to create test for this new page, what do you think?

Reply:
So far we're creating tests just for the components, for the pages, I think when one section is all done @stepsen89 will go through the testing.